.github/workflows/main.yml

```diff
@@ -2,9 +2,11 @@ name: lf-repository-api-client-dotnet-CI
 
 on:
   push:
-    branches: [ '\d+.x' ]
+    branches:
+      - v1
```

Comment:
Yeah, that's a good question. I deliberately want to hard code it because I think we want to make sure that this CI is really only for v1 and we don't want to be flexible about it. A lot of the workflow examples put the branch name explicit instead of using globing patterns.